src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateClaimSpecCallbackHandlerTest.java

```diff
@@ -1462,7 +1460,6 @@ void shouldAssignCaseName2v1_whenCaseIs2v1GlobalSearchEnabled() {
         @Test
         void shouldAssignCaseName1v1_whenCaseIs1v1GlobalSearchEnabled() {
```

Comment:
Remove GlobalSerach int name as not relevant
ie shouldAssignCaseName1v1_whenCaseIs1v1()

Reply:
Test name updated